psydac/mapping/discrete.py

```diff
@@ -11,7 +11,7 @@
 import yaml
 import yamlloader
 
-from psydac.mapping.basic import Mapping
+from sympde.topology.mapping  import Mapping
```

Comment:
Please add an empty line after this one in order to separate the external libraries from the internal library imports.